index.html

```diff
@@ -25,7 +25,165 @@
         <h1>I'm Andrea.<br>Glad to see you!</h1>
         <h2>I'm Andrea.<br>Glad to see you!</h2>
         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-          <div class="enlaces">
+        <div class="enlaces">
+          <p>Let's conect</p>
+          <ul>
+            <li>
+              <a href=""><img src="Img/twitter.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/Linkedin icon.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/medium.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/github.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/angellist.svg" alt=""></a>
+            </li>
+          </ul>
```

Comment:
[OPTIONAL]

- [ ] I recommend that you add a simple description at the `<alt>` attribute. That way your project will become more accessible :rocket:.